api/v1beta1/virtualdc_types.go

```diff
@@ -25,12 +25,10 @@ import (
 type VirtualDCSpec struct {
 	// Neco branch is a target branch name for dctest
 	//+kubebuiler:validation:Optional
-	//+kubebuilder:default=main
 	NecoBranch string `json:"necoBranch,omitempty"`
 
 	// Neco apps branch is a target branch name for dctest
```

Comment:
It would be nice to have an explanation of the default values in the comments.  `NecoBranch` too.